Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "mapbox-navigation-ios",
+    platforms: [.iOS(.v9)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "MapboxCoreNavigation",
+            targets: ["MapboxCoreNavigation"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/mapbox/mapbox-accounts-ios/", .branch("1ec5-v2.3.1")),
+        .package(url: "https://github.com/mapbox/mapbox-directions-swift/", from: "1.2.0-alpha.3"),
+        .package(url: "https://github.com/mapbox/mapbox-events-ios/", from: "0.10.6"),
+        .package(url: "https://github.com/mapbox/mapbox-navigation-native-ios/", from: "28.1.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "MapboxCoreNavigation",
+            dependencies: [
+                "MapboxAccounts",
+                "MapboxDirections",
+                "MapboxMobileEvents",
+                "MapboxNavigationNative",
+            ],
+            path: "MapboxCoreNavigation"),
```

Comment:
This is just a workaround for the fact that I haven’t moved the sources into Sources/ and the tests into Tests/.

The presence of the Info.plist file and a few Objective-C files in the MapboxCoreNavigation folder triggers this error:

> found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
> /path/to/mapbox-navigation-ios/MapboxCoreNavigation/Info.plist
> target at '/path/to/mapbox-navigation-ios/MapboxCoreNavigation' contains mixed language source files; feature not supported